massa-models/src/config/constants.rs

```diff
@@ -131,8 +131,10 @@ pub const MAX_ASYNC_MESSAGE_DATA: u64 = 1_000_000;
 pub const OPERATION_VALIDITY_PERIODS: u64 = 10;
 /// cycle duration in periods
 pub const PERIODS_PER_CYCLE: u64 = 128;
-/// cycle duration in periods
-pub const PERIODS_BETWEEN_BACKUPS: u64 = 128;
+/// Number of periods between two backups
+pub const PERIODS_BETWEEN_BACKUPS: u64 = 12800; // 100 cycles
```

Comment:
Instead of setting a const value and add a comment, you could directly do `PERIODS_PER_CYCLE * 100` or something of the sort. (just sugar, up to you)